packages/grid/_modules_/grid/components/toolbar/GridFilterToolbarButton.tsx

```diff
@@ -11,9 +13,24 @@ import {
 import { gridPreferencePanelStateSelector } from '../../hooks/features/preferencesPanel/gridPreferencePanelSelector';
 import { GridPreferencePanelsValue } from '../../hooks/features/preferencesPanel/gridPreferencePanelsValue';
 import { optionsSelector } from '../../hooks/utils/optionsSelector';
+import { GridTranslationKeys } from '../../models/api/gridLocaleTextApi';
 import { GridApiContext } from '../GridApiContext';
 
-export const GridFilterToolbarButton: React.FC<{}> = () => {
+const useStyles = makeStyles(
+  (theme) => ({
+    list: {
+      margin: theme.spacing(1, 1, 0.5),
+      padding: theme.spacing(0, 1),
+    },
+  }),
+  { name: 'MuiDataGridFilterToolbarButton' },
+);
+
+export const GridFilterToolbarButton = React.forwardRef<
+  HTMLButtonElement,
+  React.HTMLAttributes<HTMLButtonElement>
```

Comment:
This type can be more specific

```suggestion
  ButtonProps
```

Reply:
Updated